utils/spdx/src/main/kotlin/SpdxSimpleLicenseMapping.kt

```diff
@@ -28,37 +28,39 @@ import com.fasterxml.jackson.module.kotlin.readValue
  */
 object SpdxSimpleLicenseMapping {
     /**
-     * The raw map which associates custom license IDs with their corresponding SPDX license ID.
+     * The map of simple license names associated with their corresponding [SPDX license][SpdxLicense].
      */
-    internal val customLicenseIdsMap by lazy {
+    internal val simpleLicenseMapping by lazy {
         val resource = checkNotNull(javaClass.getResource("/simple-license-mapping.yml"))
         yamlMapper.readValue<Map<String, SpdxLicense>>(resource)
     }
 
     /**
-     * The map of custom license ids associated with their corresponding SPDX expression.
+     * The map of simple license names associated with their corresponding [SPDX expression][SpdxLicenseIdExpression].
      */
-    internal val customLicenseIds = customLicenseIdsMap.mapValues { (_, v) -> v.toExpression() }
-        .toSortedMap(String.CASE_INSENSITIVE_ORDER)
+    val simpleExpressionMapping by lazy {
+        simpleLicenseMapping.mapValuesTo(sortedMapOf(String.CASE_INSENSITIVE_ORDER)) { (_, v) -> v.toExpression() }
+    }
 
     /**
-     * The map of deprecated SPDX license ids associated with their current SPDX expression.
+     * The map of deprecated SPDX license IDs associated with their current [SPDX expression]
+     * [SpdxSingleLicenseExpression].
```

Comment:
Does this link work correctly with the line break in between?

Reply:
Yes (see the syntax highlighting):

![image](https://github.com/user-attachments/assets/a42db766-a211-428e-9b32-e732989fed0e)